CAIPs/caip-27.md

```diff
@@ -12,7 +12,7 @@ requires: 2, 25, 171, 217
 
 ## Simple Summary
 
-CAIP-27 defines a generic JSON-RPC method for routing method calls to a context
+CAIP-27 defines a JSON-RPC method for requesting methods to a context
```

Comment:
maybe the simple summary should be more explicit about the directionality, i.e.

```suggestion
CAIP-27 defines a JSON-RPC method for a wallet-connected application to request that wallet execute an individual method call in a specified context
```

(I think the genitive in the title can be ambiguous, i.e., is the wallet requesting or is this for making requests TO wallets)